### PR TITLE
v8.2.0 manual

### PR DESCRIPTION
**Pull request by thewchan**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

---

Closes #12

### CODE REVIEW

**Comment by thewchan** [comment]
@conda-forge-admin, please rerender

**Comment by thewchan** [comment]
@fcollonval @kir0ul review and merge request, thanks!

File: recipe/meta.yaml [hunk 1]
```diff
@@ -32,14 +32,16 @@ requirements:
...
-    - python >=3.8
+    - python >=3.8,<4.0
     - pyyaml >=5.3.1
     - pyyaml-include >=1.2
     - questionary >=1.8.1
-    - git >=2.27
+    - typing-extensions >=3.7.4,<5.0.0
+
+    # questionary 2.0.0 has requirement prompt_toolkit<=3.0.36,>=2.0, but you have prompt-toolkit 3.0.39.
+    - prompt_toolkit <=3.0.36,>=2.0
```
Comment (fcollonval) [inline recipe/meta.yaml:44]:
```suggestion
```

Could you open an PR to fix this upstream at https://github.com/conda-forge/questionary-feedstock
as we should avoid adding indirect dependency constraints to reduce maintenance and ease the env solver job.

**Review by fcollonval — CHANGES_REQUESTED** [review]
Thanks @thewchan 

Would you mind opening a fix upstream for `prompt_toolkit`?

File: recipe/meta.yaml (hunk 1 above)
Comment (fcollonval) [reply recipe/meta.yaml:44]: Fixed upstream by https://github.com/conda-forge/questionary-feedstock/pull/7

File: conda-forge.yml [hunk 2]
```diff
@@ -5,4 +5,4 @@ conda_forge_output_validation: true
 conda_build:
   pkg_format: '2'
 bot:
-  automerge: true
+  inspection: hint-all
```
Comment (fcollonval) [inline conda-forge.yml:8]: Please don't take initiative to change recipe configuration in upgrade PR except if needed.

```suggestion
  automerge: true
  inspection: hint-all
```

File: recipe/meta.yaml [hunk 3]
```diff
@@ -32,14 +32,13 @@ requirements:
...
-    - pydantic >=1.10.2,<2
+    - pydantic >=2.0.3
     - pygments >=2.7.1
-    - python >=3.8
+    - python >=3.8,<4.0
     - pyyaml >=5.3.1
     - pyyaml-include >=1.2
     - questionary >=1.8.1
-    - git >=2.27
+    - typing-extensions >=3.7.4,<5.0.0
```
Comment (fcollonval) [inline recipe/meta.yaml:41]: Same here - git is available on conda (not on PyPI). Please don't change pure conda dependencies in upgrade PR except if needed - these changes must be discussed as they may break consumer workflow.

```suggestion
    - git >=2.27
    - typing-extensions >=3.7.4,<5.0.0
```

**Review by fcollonval — APPROVED** [review]

**Pull request merged**